src/Components/Menu/PastryDishes.tsx

```diff
@@ -21,17 +22,18 @@ const menu = [
     }
 ]
 
-export default function MenuPastries() {
+export default function PastryDishes() {
     return (
         <div className="pastries-section">
             <div className="pastries-header">
                 <p className="pastries-title">Pastries & Drinks</p>
                 <p className="pastries-title-desc">This is a section of pastries & drinks. Choose your Pastries & drinks</p>
+                <img className="avocado" src={avocado} alt="avocado" />
             </div>
             <div className="pastry-menu">
                 <img className="pastry-items" src={pastries} alt="starters"></img>
                 <div className="menu">
-                    {menu.map((item: any, index: number) => (
+                    {menu.map((item: {}, index: number) => (
                         <MenuDisplay obj={item} key={index} />
```

Comment:
A common component can be written for this which can be used for Starter, Main and Pastries

Reply:
common component menudisplay is being used to render the menu in starters, mains and pastries